src/OnboardingSPA/utils/api-queuer/api-queue-executor.js

```diff
@@ -0,0 +1,23 @@
+/* eslint no-console: ["error", { allow: ["error"] }] */
+// This Executer is responsible to execute API requests in a sequence
+const apiQueueExecutor = async ( requests ) => {
+	const items = requests;
+
+	const dequeue = async ( retryCount = 1 ) => {
+		// Queue Empty
+		if ( ! items[ 0 ] ) return;
+
+		await items[ 0 ][ 1 ]()
+			.then( ( e ) => {
+				if ( e.error && retryCount < 2 ) {
```

Comment:
```suggestion
				if ( e.error && retryCount < 2) {
```
Should we move `2` to a retry count constant?

Reply:
Sure Done